frontend/public/components/factory/details.tsx

```diff
@@ -18,7 +20,7 @@ export type FirehoseResource = {
   prop: string;
 };
 
-export const DetailsPage: React.SFC<DetailsPageProps> = (props) => <Firehose resources={[{
+export const DetailsPage = withFallback<DetailsPageProps>((props) => <Firehose resources={[{
```

Comment:
Need to make sure you access the wrapped component [in the unit test](https://github.com/openshift/console/blob/master/frontend/__tests__/components/factory/details.spec.tsx#L17).